requirements/prod.txt

```diff
@@ -126,12 +126,12 @@ bleach==3.1.0 \
 boto3==1.9.230 \
     --hash=sha256:972050c8a36daecd1f4eb8689cca3048b67d3c9031db067471a936e8fe839d14 \
     --hash=sha256:c96be2c71ed4893df29f0ce11cc3562109d9c14de6d7a1bd9f82c3356303fdf4
-celery==4.1.1 \
-    --hash=sha256:6fc4678d1692af97e137b2a9f1c04efd8e7e2fb7134c5c5ad60738cdd927762f \
-    --hash=sha256:d1f2a3359bdbdfb344edce98b8e891f5fe64f8a11c5a45538ec20ac237c971f5 # pyup: <4.2
 botocore==1.12.230 \
     --hash=sha256:18b7064b4bc6f7cb295aa48917eb91efe817cc0111ffbcb6559f50da0cee599d \
     --hash=sha256:30ff956bfef7dd91f35235ac705f9457f575071c2a577a01f8a2cf19eea95d02
+celery==4.3.0 \
+    --hash=sha256:4c4532aa683f170f40bd76f928b70bc06ff171a959e06e71bf35f2f9d6031ef9 \
+    --hash=sha256:528e56767ae7e43a16cfef24ee1062491f5754368d38fcfffa861cdb9ef219be
```

Comment:
gah! the celery dependency was in the middle of the boto updates for a reason - otherwise each daily boto3 update causes a merge conflict with the accompanying botocore dep.